benchmark/benchmarks/krausest/lib/components/Application.ts

```diff
@@ -1,6 +1,9 @@
-import { swapRows, type Item, updateData, buildData } from '@/utils/data';
```

Comment:
Throughout the benchmarks, use [subpath
imports](https://nodejs.org/api/packages.html#subpath-imports) (the new npm standard) rather than
path mappings.